pyaerocom/trends_helpers.py

```diff
@@ -80,6 +88,12 @@ def _get_unique_seasons(idx):
             seasons.append(seas)
     return seasons
 
+def _get_season_from_months(months: str) -> str:
+    if months in MONTHS_CODES.keys():
+        return MONTHS_CODES[months]
+    else:
+        raise ValueError(f"{months} is not a valid season")
```

Comment:
```suggestion
    if months in MONTHS_CODES.keys():
        return MONTHS_CODES[months]
    raise ValueError(f"{months} is not a valid season")
```
else is not really needed